pkgs/development/python-modules/django-simple-captcha/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, buildPythonPackage, mock, 
+django, django-ranged-response, pillow }:
+
+buildPythonPackage rec {
+  pname = "django-simple-captcha";
+  version = "0.5.6";
+
+  src = fetchurl {
+    url = "https://files.pythonhosted.org/packages/4d/46/44aff307e370e873ebb44dd8e9a1bcde10ddd1e55779361a5654d273d939/django-simple-captcha-0.5.6.zip";
+    sha256 = "02sbbsjlxa445hdfpxj4wmy1ynfj1vj0xblqhs7r05a2pbl3dzfn";
+  };
+
+  doCheck = false;
+
+  buildInputs = [ mock ];
```

Comment:
Again, I can't see a need for this, and in fact _having_ it obscures the fact that it actually depends on `six`, which it's only seeing incidentally as it's brought in by this `mock` dependency - but of course it isn't `propagated` so it won't get picked up as a proper dependency of the nix package.

Reply:
Thanks a lot for the explanation!